server/service/hosts.go

```diff
@@ -719,8 +746,15 @@ func (svc *Service) DeleteHost(ctx context.Context, id uint) error {
 		return ctxerr.Wrap(ctx, err, "delete host")
 	}
 
-	if host.Platform == "darwin" {
-		return svc.maybeRestorePendingDEPHost(ctx, host)
+	if host.Platform == "windows" || host.Platform == "darwin" {
```

Comment:
all of the magic strings for "windows" and "darwin" make me nervous, would love to see them as constants that we import and use the variables so we know if we mis-spell it won't compile :D but I won't hold that up here. maybe an eng-initiated story to update them all at a later date.

Reply:
agreed, and they're all across the app! we can also just start using two constants from now on and update the existing occurrences later